ui/lib/apps/InstanceProfiling/pages/Detail.tsx

```diff
@@ -10,11 +10,29 @@ import { CardTable, DateTime, Head, Descriptions } from '@lib/components'
 import { useClientRequestWithPolling } from '@lib/utils/useClientRequest'
 import { InstanceKindName } from '@lib/utils/instanceTable'
 import useQueryParams from '@lib/utils/useQueryParams'
+import { ScrollablePane } from 'office-ui-fabric-react'
+import { MenuInfo } from 'rc-menu/lib/interface'
 
-function mapData(data) {
+enum ViewOutputTypeOptions {
+  FlameGraph = 'instance_profiling.detail.table.columns.selection.flamegraph',
+  Graph = 'instance_profiling.detail.table.columns.selection.graph',
+}
```

Comment:
I think you can use a simple enum, instead of mixing i18n with it. I18N should happen only at the presentation layer, instead of in the models:

```ts
// Define the simplist model value. As a model, it is relied by others. It should do its best without knowing any other outside usages.
enum ViewOutputTypeOptions {
  FlameGraph,
  Graph,
}

// Use the model value in models:
task.view_outputs = [ViewOutputTypeOptions.FlameGraph, ViewOutputTypeOptions.Graph]

// Use the single model value at the presentation layer to support I18N:
    { task.view_outputs.map(vo => (
        <Menu.Item key={vo}>
          {t('instance_profiling.detail.table.columns.selection.view', {
            type: {t(`instance_profiling.detail.table.columns.selection.types.${vo}`)}
          })}
        </Menu.Item>
    )) }
```

Reply:
Maybe string enums is better
```
enum ViewOutputTypeOptions {
  FlameGraph = 'flamegraph', 
  Graph = 'graph',
}
```